detekt-baseline.xml

```diff
@@ -688,5 +688,6 @@
     <ID>UnusedPrivateProperty:UserService.kt$UserService$private val userTransformer: UserTransformer</ID>
     <ID>UnusedPrivateProperty:WorkingDayCountService.kt$WorkingDayCountService$i</ID>
     <ID>VariableNaming:ApplicationServiceTest.kt$ApplicationServiceTest.SubmitApplication$val _schema = application.schemaVersion as Cas2ApplicationJsonSchemaEntity</ID>
+    <ID>LargeClass:BedUtilisationReportGeneratorTest.kt$BedUtilisationReportGeneratorTest</ID>
```

Comment:
Going forward (don't sorry about it now), can we please use @SuppressWarning instead of adding entries into this file? This makes the exclusions more visible and (hopefully) more likely to be fixed

Reply:
Thanks good to know for the future